core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterDMTest.java

```diff
@@ -12274,6 +12274,62 @@ private static RelDataType createUserDefinedType(RelDataTypeFactory typeFactory)
     assertThat(toSql(root, DatabaseProduct.CALCITE.getDialect()), isLinux(expectedQuery));
   }
 
+  @Test
+  public void testCURRENT_JOB_IDFunc() {
```

Comment:
change testcase name to testCURRENT_JOB_IDFunction.

Reply:
Done changes.